app/views/shared/_progress_steps.html.slim

```diff
@@ -0,0 +1,8 @@
+- active ||= 0
```

Comment:
what if you also did `num_steps ||= 6` in case we want to have a variable # of steps?